.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: Akeneo Coupling Detector CI
+on: [push]
+jobs:
+    build_and_test:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php-versions: ['7.2', '7.3', '7.4', '8.0']
```

Comment:
I would remove 7.2 and 7.3, not maintained anymore

Reply:
Indeed, but I also would like to remove them from the composer.json to "officially" not support them anymore.

Reply:
And yet, we still use the coupling on PIM 4.0, which still uses 7.3 :no_mouth:

Reply:
Just tag it :)